src/script/auth/page/Login.tsx

```diff
@@ -192,9 +193,8 @@ const LoginComponent = ({
 
       return navigate(ROUTE.HISTORY_INFO);
     } catch (error) {
-      if ((error as BackendError).label) {
-        const backendError = error as BackendError;
-        switch (backendError.label) {
+      if (isBackendError(error)) {
```

Comment:
ended up using isBackendError predicate, this way we're sure we are dealing with error that contains label